src/validations/test/rules/ssp.xspec

```diff
@@ -698,32 +997,63 @@
                     </x:context>
                     <x:expect-assert
                         id="extraneous-response-description"
-                        label="it generates a warning."
-                        role="warning" />
+                        label="this is invalid."
+                        role="error" />
+                </x:scenario>
+                <x:scenario
+                    label="and implemented requirements do not have explanatory statement descriptions defined directly in the statement">
+                    <x:context>
+                        <system-security-plan
+                            xmlns="http://csrc.nist.gov/ns/oscal/1.0">
+                            <control-implementation>
+                                <implemented-requirement>
+                                    <statement/>
+                                </implemented-requirement>
+                            </control-implementation>
+                        </system-security-plan>
+                    </x:context>
+                    <x:expect-not-assert
+                        id="extraneous-response-description"
+                        label="this is valid."
+                        role="error" />
                 </x:scenario>
                 <x:scenario
                     label="and implemented requirements have explanatory statement remarks defined directly in the statement">
                     <x:context>
                         <system-security-plan
                             xmlns="http://csrc.nist.gov/ns/oscal/1.0">
-                            <system-characteristics>
-                                <security-sensitivity-level>fips-199-low</security-sensitivity-level>
-                            </system-characteristics>
                             <control-implementation>
-                                <implemented-requirement
-                                    control-id="ac-2"
-                                    uuid="4325e92d-e888-42ff-9d49-f5231129c4d3">"&gt; <statement
-                                        statement-id="ac-2_stmt.a"
-                                        uuid="a8419fd8-2666-4851-8191-94d054c9ef1c">"&gt; <remarks>
-                                            <p>This shouldn not be here.</p>
-                                        </remarks></statement></implemented-requirement>
+                                <implemented-requirement>
+                                    <statement>
+                                        <remarks>
+                                            <p>This shoud not be here.</p>
```

Comment:
Typo.

Reply:
This does not exist in the ssp.xspec.  You may not be looking at the most recent commit.

Reply:
I see the letters "shoud".